src/main/java/uk/gov/mca/beacons/service/model/BeaconUse.java

```diff
@@ -56,10 +56,10 @@
 
   private Boolean mobileTelephone;
 
-  @Column(name = "mobile_telephone_1")
+  @Column(name = "mobile_telephone_input_1")
   private String mobileTelephoneInput1;
```

Comment:
`_input_` Isn't this naming residue from the frontend?  We only called it `..._input` to differentiate the text input element from the checkbox element.  I'm thinking we shouldn't let frontend concerns bleed into our backend model.